.github/integ-config/integ-all.yml

```diff
@@ -7,39 +7,68 @@ extended_browser_list: &extended_browser_list
-  - dynamic-user-pool-groups-default dynamic-user-pool-groups-default
-  # - dynamic-user-pool-groups-owner-based-default dynamic-user-pool-groups-owner-based-default
-  - private-auth-default private-auth-default
-  - private-auth-iam private-auth-iam
-  - public-auth-default public-auth-default
-  - public-auth-iam public-auth-iam
-  - owner-custom-claim-default owner-custom-claim-default
-  - owner-custom-field-default owner-custom-field-default
-
-datastore_auth_scenarios_sample_spec_v2:
-  &datastore_auth_scenarios_sample_spec_v2
-  - v2/owner-based-default-v2 owner-based-default # TODO: Add `owner-based-operations` when tests pass
-  - v2/static-user-pool-groups-default-v2 static-user-pool-groups-default
-  - v2/static-user-pool-groups-operations-v2 static-user-pool-groups-operations
-  - v2/owner-and-group-different-models-default-v2 owner-and-group-different-models-default
-  - v2/owner-and-group-same-model-default-v2 owner-and-group-same-model-default
-  - v2/owner-and-group-same-model-operations-v2 owner-and-group-same-model-operations
-  - v2/private-auth-iam-v2 private-auth-iam
-  - v2/public-auth-default-v2 public-auth-default
-  - v2/public-auth-iam-v2 public-auth-iam
-  - v2/owner-custom-claim-default-v2 owner-custom-claim-default
-  - v2/owner-custom-field-default-v2 owner-custom-field-default
-  - v2/dynamic-user-pool-groups-default-v2 dynamic-user-pool-groups-default
-  - v2/private-auth-default-v2 private-auth-default
+datastore_auth_scenarios: &datastore_auth_scenarios
+  # TODO: Add `owner-based-operations` when tests pass
+  - sample_name: owner-based-default #39
+    spec: owner-based-default
+  - sample_name: static-user-pool-groups-default #62
+    spec: static-user-pool-groups-default
+  - sample_name: static-user-pool-groups-operations #68
+    spec: static-user-pool-groups-operations
+  - sample_name: owner-and-group-different-models-default #80
+    spec: owner-and-group-different-models-default
+  - sample_name: owner-and-group-same-model-default #68
+    spec: owner-and-group-same-model-default
+  - sample_name: owner-and-group-same-model-operations #74
+    spec: owner-and-group-same-model-operations
+  - sample_name: dynamic-user-pool-groups-default #64
+    spec: dynamic-user-pool-groups-default
+  - sample_name: private-auth-default #40
+    spec: private-auth-default
+  - sample_name: private-auth-iam #32
+    spec: private-auth-iam
+  - sample_name: public-auth-default #38
+    spec: public-auth-default
+  - sample_name: public-auth-iam #30
+    spec: public-auth-iam
+  - sample_name: owner-custom-claim-default #52
+    spec: owner-custom-claim-default
+  - sample_name: owner-custom-field-default #52
+    spec: owner-custom-field-default
   # TODO: The below test fails and ends up failing all the other tests in this group
-  # - v2/dynamic-user-pool-groups-owner-based-default-v2 dynamic-user-pool-groups-owner-based-default
+  # - sample_name: dynamic-user-pool-groups-owner-based-default #88
```

Comment:
Running this test seems to crash gh actions and stops all others jobs from running as well. Will need to investigate further. Commenting for now
[Please see sample run for reference](https://github.com/stocaaro/amplify-js/actions/runs/5472416024)